src/java.desktop/windows/native/libawt/windows/awt_Menu.h

```diff
@@ -73,7 +73,7 @@ class AwtMenu : public AwtMenuItem {
 
     /*for multifont menu */
     BOOL IsTopMenu();
-    virtual AwtMenuItem* GetItem(jobject target, long index);
+    virtual AwtMenuItem* GetItem(jobject target, int index);
```

Comment:
Hi @aivanov-jdk are you OK leaving this inconsistent with the definition?
https://github.com/openjdk/jdk/blob/16b5a91461db1765e2e7596ebaaf1299cec9b0c8/src/java.desktop/windows/native/libawt/windows/awt_Menu.cpp#L261

Reply:
The declaration and implementation have to match.

Reply:
To minimise the number of changes, we can go for using `jint` in `AwtMenu::GetItem`.

What do you thing, @djelinski and @TheShermanTanker?

Reply:
Hmm, I lean towards jint as I feel it conveys the fact that it is a Java parameter clearer, intuitively to me it makes sense that a Java integer type would still work in a for loop

Reply:
You're right… it gives a hint it'll be an upcall into Java. Let's go for `jint` then.

I don't think there's a need to change the type of the for-loop variable to `jint`.

Reply:
Oh no, I didn't mean to change the loop variable, rather that leaving the jint as is should be fine in the for loop